RepoDb.Core/RepoDb/Extensions/DbCommandExtension.cs

```diff
@@ -340,6 +346,12 @@ private static void EnsureTableValueParameter(IDbDataParameter parameter)
                     dbType = Converter.EnumDefaultDatabaseType;
                 }
 
+                // DbType fallback
+                if (dbType == null)
+                {
+                    dbType = TypeMapper.GetFallback(classProperty.PropertyInfo.PropertyType);
```

Comment:
Same here, the line [333](https://github.com/mikependon/RepoDB/blob/5472110ac13c9b816f9cc77aaa87e54cf706c4c7/RepoDb.Core/RepoDb/Extensions/DbCommandExtension.cs#L333) should do it already.